frontend/src/App.js

```diff
@@ -1,23 +1,23 @@
 import './App.css';
 import styled from 'styled-components/macro';
 import AppHeader from "./components/AppHeader";
-import ProductTable from "./components/ProductTable";
+import ShowProductsInTable from "./components/ProductTable";
 
 function App() {
-  return (
-      <PageLayout>
-        <AppHeader/>
-        <ProductTable/>
-      </PageLayout>
-  );
+    return (
+        <PageLayout>
+            <AppHeader/>
+            <ShowProductsInTable/>
```

Comment:
This sounds like a (helper) function, use a more subject oriented naming scheme for components. Imho, `ProductTable` was fine.